pmga/3.0.2/Dockerfile

```diff
@@ -0,0 +1,48 @@
+# Base Image
+FROM ubuntu:focal
+
+ARG PMGA_VERSION="3.0.2"
+ARG MASH_VERSION="1.1"
+
+# Metadata
+LABEL base.image="ubuntu:focal"
+LABEL version="1"
```

Comment:
Please change to `LABEL version="1"` to `LABEL dockerfile.version="1"`

explanation: https://github.com/StaPH-B/docker-builds/issues/118

Reply:
Okie doke, copied from another one. I'll try to find that one again